automation/patch_igz/patch_env_template.yml

```diff
@@ -33,3 +33,6 @@ REGISTRY_USERNAME:
 
 # Optional - will attempt docker login if defined
 REGISTRY_PASSWORD:
+
+# Optional - only required if asking to reset DB
+DB_USER:
```

Comment:
Why do you need a user? We always deploy MLRun DB with the `root` user, and I think it's the only valid user (we don't create other users for the DB). Why not always use `root`?